stack-really-old.yaml

```diff
@@ -26,3 +26,5 @@ extra-deps:
 - code-page-0.2@sha256:f701393cb1ff7b3ec7880816abc44387647811be31670e884e02d6a20c4aa508,2356
 - fail-4.9.0.0@sha256:35d1ee29364447c1b7d616bb1ee31f162b73e85fea91d7ca6441cf901398f572,1051
 - tasty-expected-failure-0.11.1.2@sha256:ff9bad876ec28415e8fb6837362f2086331cf23d7f1468536aaabd91d53f66e4,1826
+- gauge-0.2.5@sha256:8d60450bdec985c146d5632d4f5a8d60cec27d71ba6787ed1ee64d945d4c7c33,3923
+- time-1.10@sha256:536801b30aa2ce66da07cb19847827662650907efb2af4c8bef0a6276445075f,5738
```

Comment:
It's not clear to me by which criteria this list is structured and ordered, so I just added these two missing extra-deps at the end ...